frontend/src/components/homepage/homepage.tsx

```diff
@@ -1,6 +1,6 @@
-import Timer from "./timer";
+import Timer from "./Timer";
 import { Separator } from "../ui/separator";
-import Testimonials from "./testimonials";
+import Testimonials from "./Testimonials";
 import Events from "./events";
```

Comment:
Same thing goes for this since the file name is in lower case.
![Screenshot 2024-05-23 at 7 17 57 PM](https://github.com/Thangnguyenvn98/GatheringGlobe/assets/69278249/5ab752cb-57dd-45a9-af40-67beb7fd20ad)